tests/sentry/api/endpoints/issues/test_related_issues.py

```diff
@@ -47,20 +45,43 @@ def test_same_root_related_issues(self) -> None:
         assert response.json() == {"type": "same_root_cause", "data": [5], "meta": {}}
 
     def test_trace_connected_errors(self) -> None:
-        error_event, _, another_proj_event = self.load_errors(self.project, uuid4().hex[:16])
+        error_event, _, another_proj_event = self.load_errors(self.project)
         group = error_event.group
-        self.group_id = error_event.group_id  # type: ignore[assignment]
         recommended_event = group.get_recommended_event_for_environments()  # type: ignore[union-attr]
         assert recommended_event is not None  # It helps with typing
 
         assert error_event.group_id != another_proj_event.group_id
         assert error_event.project.id != another_proj_event.project.id
         assert error_event.trace_id == another_proj_event.trace_id
 
+        # This sets the group_id to the one we want to query about
+        self.group_id = error_event.group_id  # type: ignore[assignment]
         response = self.get_success_response(qs_params={"type": "trace_connected"})
         assert response.json() == {
             "type": "trace_connected",
             # This is the other issue in the trace that it is not itself
             "data": [another_proj_event.group_id],
             "meta": {"event_id": recommended_event.event_id, "trace_id": error_event.trace_id},
         }
+
+    def test_trace_connected_errors_specific_event(self) -> None:
+        error_event, _, another_proj_event = self.load_errors(self.project)
+
+        # This sets the group_id to the one we want to query about
+        self.group_id = another_proj_event.group_id  # type: ignore[assignment]
+        response = self.get_success_response(
+            qs_params={
+                "type": "trace_connected",
+                "event_id": another_proj_event.event_id,
+                "project_id": another_proj_event.project_id,
```

Comment:
We test in here that we pass `event_id` and `project_id` unlike the previous test.